src/structures/BaseGuildVoiceChannel.js

```diff
@@ -68,8 +68,13 @@ class BaseGuildVoiceChannel extends GuildChannel {
    */
   get joinable() {
     if (!this.viewable) return false;
-    if (!this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false)) return false;
-    return true;
+    const permissions = this.permissionsFor(this.client.user);
+    if (!permissions) return false;
+
+    // This flag allows joining even if timed out
+    if (permissions.has(Permissions.FLAGS.ADMINISTRATOR, false)) return true;
+    if (!permissions.has(Permissions.FLAGS.CONNECT, false)) return false;
+    return this.guild.me.communicationDisabledUntilTimestamp < Date.now();
```

Comment:
```suggestion
    const { communicationDisabledUntilTimestamp } = this.guild.me;
    return communicationDisabledUntilTimestamp ? communicationDisabledUntilTimestamp < Date.now() : true;
```
If `communicationDisabledUntilTimestamp` will be `null` it will return `false`, but if `communicationDisabledUntilTimestamp` is `null` you aren't timeouted and can join the channel.

Reply:
Or
```suggestion
    return this.guild.me.communicationDisabledUntilTimestamp ?? 0 < Date.now();
```

Reply:
I'm confused by these suggestions... can you explain why this change is needed?

Reply:
They're not needed at all, `null < Date.now()` is always executed as `0 < Date.now()` because the `<` operator converts both expressions to a number primitive, and since `null` has neither `@@toPrimitive` nor `toValue`... it falls back to default behavior: `Number(null)`, which is effectively `0`.

So doing `null < Date.now()` is equivalent to doing `0 < Date.now()`.

Reply:
Sorry, my suggestion was wrong, but you should still consider @Syjalo's suggestion because otherwise, this will return false when a user is not timed out, which is misleading